src/utility.jl

```diff
@@ -74,3 +74,115 @@ function visualize_flow(method::ColorBased, flow::Array{SVector{2, Float64}, 2};
     return hsv
     
 end
+
+struct Point{T}
```

Comment:
Do we need to introduce the `Point` type?  Aren't we representing points and flow vectors with `SVector{2, Float64}`?

Reply:
I was also thinking that there is no need of the `Point` struct. Next commit will take care of it.